cnf-certification-test/platform/suite.go

```diff
@@ -114,8 +118,34 @@ var _ = ginkgo.Describe(common.PlatformAlterationTestKey, func() {
 			ginkgo.Skip(" non ocp cluster ")
 		}
 	})
+
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestServiceMeshIdentifier)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.DebugPods)
+		TestServiceMesh(&env)
+	})
 })
 
+func TestServiceMesh(env *provider.TestEnvironment) {
+	var badPods []string
+	for _, put := range env.Pods {
+		flag := 0
+		data := put.Containers
+		for _, d := range data {
+			if d.Status.Name == istio {
```

Comment:
Do we want to use some sort of `string.Contains` here?  Is it always just `istio` the string?  What about other types of service mesh?  We could potentially keep a slice `[]string{"istio", "other service meshes", "etc"}` for example to loop through in case we need to add more later.